melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/helpers/LandkodeMapper.java

```diff
@@ -47,7 +47,7 @@ public static String finnLandkodeIso3(String landkodeIso2, boolean ukjentLandkod
             .filter(entry -> entry.getValue().equals(landkodeIso2))
             .map(Map.Entry::getKey)
             .findFirst()
-            .orElse(ukjentLandkodeMedPdlFormat ? "XUK" : "???");
+            .orElse(skalReturnereNullForUkjent ? null : "XUK");
```

Comment:
Husk at vi bruker "???" i API. Dvs. hvis vi skal gjøre XUK til ukjent også for oss, må vi endre i api også.